org.sdrangel.SDRangel.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: org.sdrangel.SDRangel
 runtime: org.kde.Platform
-runtime-version: 5.15-21.08
+runtime-version: 5.15-22.08
 sdk: org.kde.Sdk
 base: io.qt.qtwebengine.BaseApp
```

Comment:
This PR probably needs to wait until the `io.qt.qtwebengine.BaseApp` gets updated to 5.15-22.08.